caffeinated/admin/extend/registration_form/templates/question_groups_main_meta_box.template.php

```diff
@@ -192,6 +192,27 @@ class="regular-text"<?php echo $disabled; ?>>
                                            name="question_orders[<?php echo $question_ID; ?>]"
                                            value="<?php echo $question_order; ?>">
                                 </label>
+                                <?php
+                                if (EE_Registry::instance()->CAP->current_user_can(
+                                    'ee_edit_question',
+                                    'espresso_registration_form_edit_question',
+                                    $question->ID()
+                                )) {
+                                    $edit_query_args = array(
+                                        'action' => 'edit_question',
+                                        'QST_ID' => $question->ID(),
+                                    );
+                                    $edit_link = EE_Admin_Page::add_query_args_and_nonce($edit_query_args, EE_FORMS_ADMIN_URL);
+
+                                    echo '<a href="' . $edit_link . '" target="_blank" title="' .
+                                        esc_attr__(
+                                            'Edit Event',
+                                            'event_espresso'
+                                        )
```

Comment:
`'Edit Event'` ???  copy pasta maybe?


```suggestion
                                    echo '<a href="' . $edit_link . '" target="_blank" title="' .
                                        sprintf(
                                        	esc_attr__( 'Edit %s', 'event_espresso' ),
                                        	$question->admin_label()
                                        )
```